### PR TITLE
Fix/selfservice newticket form

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: src/CommonITILObject.php [hunk 1]
```diff
@@ -36,6 +36,7 @@
 use Glpi\RichText\RichText;
 use Glpi\Team\Team;
 use Glpi\Toolbox\Sanitizer;
+use Laminas\I18n\Validator\IsInt;
```
Comment (cedric-anne) [inline src/CommonITILObject.php:39]:
```suggestion
```

**Review by cedric-anne — APPROVED** [review]

**Comment by orthagh** [comment]
With end of morning for merge this, i want to do a small factorization

**Comment by orthagh** [comment]
Ready for review/merge

File: src/CommonITILObject.php (hunk 1 above)
Comment (trasher) [inline src/CommonITILObject.php:39]: This seems not used

**Review by trasher — APPROVED** [review]

**Pull request merged**